### DISCUSSION TITLE
Is it possible to record an Espresso UI Test when using Capacitor?

### DISCUSSION THREAD

**Discussion by uncvrd** [ionic-team/capacitor#3771] [Q&A]
Hi all, fairly new to the android side of Capacitor. I've been trying to set up UI Tests using Android Studio's Espresso. However, it doesn't seem to be logging any of my user interactions when running Espresso. Steps include:

1. `Run > Record Espresso test`
2. Interface for testing pops up and Emulator boots up
3. I click on an item on my app
4. Interaction is not recorded

Streamable video showing this problem: https://streamable.com/moz5gt

I did some preliminary digging and I remember that Ionic uses webview to display contents? Is that possibly why this does not work correctly? Interestingly enough this seems to work ok when I create tests in iOS. Any insight would be super helpful. Thanks!!

**Comment by frossigneux** [comment]
I have the same problem. Any update?

**Comment by fernando-castillo-seagull-com** [comment]
same here with cordova... when i try to add an assertion the last node that i can read is webview... maybe is related to that?
